### ISSUE TITLE
Artifacts at high resolution settings

### ISSUE THREAD

**Issue by MazeRuiZhang** [crowsonkb/style-transfer-pytorch#18]
Hi  @crowsonkb ,

I like this repository so much! I already tried some style transfers with different settings, and the results at higher resolution are very impressive! Most drawing details are vivid even at a very close look.

But the only concern is that for some results at 2560x1440 resolution, there are still minor artifacts. Those artifacts are mostly black.

My settings are as following:
-s 2560 
--content-weight 0.02 
--random-seed 1 
--iterations 600 
--devices cuda:0 
--pooling l2 
--tv-weight 0.0 
--save-every 5000  
--initial-iterations 30000 
--style-scale-fac 1.0 
--min-scale 460  
--avg-decay 0.99 
--step-size 0.02

![artifact_1](https://user-images.githubusercontent.com/23298787/162609507-b6b15b4e-9fdf-4156-9f30-c0e056c9c855.png)

To solve this artifact, I have used hundreds of hyper-parameter combos, but no luck. Based on your experience, could you share some tips to remove the artifact at a higher resolution setting?

Thank you so much!
